ql/src/java/org/apache/hadoop/hive/ql/plan/TableDesc.java

```diff
@@ -260,4 +260,12 @@ public String toString() {
         + ", outputFileFormatClass=" + outputFileFormatClass + ", properties="
         + properties + ", jobProperties=" + jobProperties + "]";
   }
+
+  public void setTableDescProperty(String key, String value) {
```

Comment:
do you really need these methods, if yes why not simply `setProperty`, `getProperty`

Reply:
Done.